### DISCUSSION TITLE
Specify dns addresses for nodes and api ingress

### DISCUSSION THREAD

**Discussion by privateVoit** [caos/orbos#818] [Ideas]
Hi there!

We are currently deploying nodes in a network with dhcp enabled. It takes the hassle out of managing addresses. But it turns out to be a massive problem in a disaster recovery case.
In such a situation, our vms would be recreated with new mac addresses at our secondary datacenter. Therefore all of the nodes would be getting new ips.

It would be cool if we could specifiy dns addresses of nodes in the orbiter.yaml. This also would mean that the api ingress ip would need a dns record and that ORBOS would use said address to join the cluster.

What are your opinions on this? Is this something you might consider?

**Comment by eliobischof** [comment]
Switching from IP's to domain names is rather hard to achieve, as ORBOS uses tools that need to be aware of the node IP's and virtual IP's, e.g. Keepalived or Firewalld. So currently, we are not planning to switch to DNS configuration.
